src/main/java/uk/ac/ebi/eva/utils/MongoDBHelper.java

```diff
@@ -31,7 +31,7 @@
 /**
  * @author Diego Poggioli
  *
- *
+ * Utility class dealing with MongoDB connections using pipelineOptions
```

Comment:
"pipeline options" instead of the variable name would allow to provide those options in other ways without making the documentation look weird.